tests/registration.spec.js

```diff
@@ -1,164 +1,128 @@
 const { expect, test } = require("@playwright/test");
 import { faker } from "@faker-js/faker";
+import BasePage from "../pageobjects/BasePage";
+import HomePage from "../pageobjects/HomePage";
+import SignUpPopup from "../components/SignUpPopup";
+import SettingsPage from "../pageobjects/SettingsPage";
+import Cabinet from "../pageobjects/Cabinet";
+import RemoveAccountPopup from "../components/RemoveAccountPopup";
 
 test.describe("As User I should be able to register on the site", () => {
+  let basepage;
+  let homepage;
+  let signUpPopup;
+  let cabinet;
+  let settings;
+  let removeAccountPopup;
+
   test.beforeEach(async ({ page }) => {
-    await page.goto("/");
+    basepage = new BasePage(page);
+    homepage = new HomePage(page);
+    signUpPopup = new SignUpPopup(page);
+    cabinet = new Cabinet(page);
```

Comment:
Cabinet is kind of abstract page I would say, it is confusing to see instance of Cabinet

Reply:
in fact you use it as GaragePage. This is the page you are being redirected to after successful registration